app/controllers/submissions_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
     elsif !@form.deployable_form? && !current_user
       redirect_to index_path, alert: 'Form is not currently deployed.'
     end
-    @form.increment!(:survey_form_activations) unless current_user
+    @form.increment!(:survey_form_activations) #unless current_user
```

Comment:
Always record an activation which can be followed by a submission, whether or not we have a logged in user ( keeps numbers from getting skewed )

Reply:
I've updated this PR to only persist the submission if the form is live.   If the form is not live and we do a submit on preview we will still get a successful process flow, just no db persistence